src/main/java/uk/gov/hmcts/reform/slc/config/FtpConfiguration.java

```diff
@@ -1,49 +1,91 @@
 package uk.gov.hmcts.reform.slc.config;
 
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-import uk.gov.hmcts.reform.slc.model.FtpConfigProperties;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.stereotype.Component;
 
-@Configuration
+@Component
```

Comment:
is this annotation necessary? i think `@ConfigurationProperties` already stands for bean

Reply:
Either we have this config or add `@EnableConfigurationProperties(FtpConfiguration.class)` on `Application` class

Reply:
yes that's required